update_readme.py

```diff
@@ -157,9 +176,36 @@
 <!-- prettier-ignore-start -->
 <!-- markdownlint-disable -->
   <tr>
-'''
-new_readme_content += "\n".join(contributor_info)
```

Comment:
there are some parts in the python script you removed please add them back and carefully check not to missout any parts

Reply:
> 

Sorry for that. Now I added that.